src/js/components/form/FormGroupContainer.js

```diff
@@ -7,10 +8,16 @@ const FormGroupContainer = (props) => {
   if (props.onRemove != null) {
     removeButton = (
       <div className="form-group-container-action-button-group">
-        <a className="button button-primary-link"
-          onClick={props.onRemove}>
-          <Icon id="close" color="grey" size="tiny" family="tiny"/>
-        </a>
+        <Tooltip content="Delete"
+          interactive={true}
```

Comment:
Let's remove the `interactive` property (it defaults to `false`), since there's nothing in the tooltip to interact with.